benchmark/index.ts

```diff
@@ -1,22 +1,19 @@
-import * as Benchmark from 'benchmark'
+import Benchmark from 'benchmark'
 
-import { solveString, solveCells, parseStringFormat, printBoard } from '../index'
-import { times } from '../lib';
+import {
+  solveString,
+  solveCells,
+  parseStringFormat,
+  printBoard
+} from '../index.js'
+import { times } from '../lib/index.js'
 
-import * as sudokuC from 'sudoku-c'
```

Comment:
You removed all the comparison benchmarks, that's not intended.
We want to compare this library against others.